tests/test_bounds.py

```diff
@@ -166,9 +166,6 @@ def test_raises_errors_for_data_dim_and_length(self):
         # If coords dimensions does not equal 1.
         with pytest.raises(ValueError):
             ds.bounds.add_bounds("Y")
-        # If coords are length of <=1.
-        with pytest.raises(ValueError):
-            ds.bounds.add_bounds("X")
```

Comment:
Do we need to test for this case: 

when a dimension with length <= 1 that does not have bounds, it does not get bounds added to the axis and a warning is thrown

Reply:
I added a test to check that the original dataset is returned (no bounds added) and suppressed the warning.